Dockerfile

```diff
@@ -13,33 +13,33 @@ ENV USER ${NB_USER}
 ENV HOME /home/${NB_USER}
 
 RUN groupadd --gid ${NB_UID} ${NB_USER}
-RUN useradd --uid ${NB_UID}  --gid ${NB_UID} ${NB_USER}
-USER ${NB_USER}
```

Comment:
why were we able to drop this line?

Reply:
ah, nvm, I see it declared later. out of curiosity, was there a reason for that?

Reply:
Fewer layers, it needed build tools installed, so I use the su to run commands as the notebook user, then uninstall the build tools, and at that point everything else can be run as the notebook user.

Reply:
ahhh, SMART. cool!